.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+### If you think this may be a potential security concern do not file an issue on GitHub. Please send an email to <?>. 
```

Comment:
Should I just put the SignalR team email or is there a higher priority list that it should be sent to?

Reply:
https://technet.microsoft.com/en-us/mt764065.aspx?f=255&MSPPError=-2147217396
secure@microsoft.com

Reply:
Yes, `secure@microsoft.com` is the place.

Reply:
How about we use the exact text from the Home repo, but put it up top as you've done here.

This is the text:

> If you believe you have an issue that affects the security of the platform please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://technet.microsoft.com/en-us/mt764065.aspx).